### PR TITLE
[Maestro] User Docs Updates

### PR DESCRIPTION
**Pull request by MitchellShiell**

### CODE REVIEW

**Comment by MitchellShiell** [comment]
@leoraba Along with the general review, take a look at the prereq page, it's a bit vague, you can leave a comment on any information you feel is missing or could add to the explanation of Kafka (what it is, and why we use it)

**Review by leoraba — CHANGES_REQUESTED** [review]

File: markdown/documentation/maestro/admin-guide/mappings.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: Managing Index Mappings
+--- 
+
+By default, Maestro builds the Elasticsearch index based on a pre-defined default index mapping. The name of this default mapping gets set in the `.env.maestro` environment variable file.
+
+While this mapping is not configurable in runtime, there may be cases where an administrator will need to change the mapping.
+
+For example, when updating Songs' <a href="/documentation/song/user-guide/schema/" target="_blank" rel="noopener noreferrer">dynamic schemas</a> the administrator must also update Maestros' index mapping.
```
[the remark this answers is not in the document]
Comment (MitchellShiell) [reply markdown/documentation/maestro/admin-guide/mappings.md:9]: Good catch, this formatting works in local but not live, I'll double check all links and update them

File: markdown/documentation/maestro/installation/configuration/index.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+---
+title: Configuring Maestro
+---
+
+The sections below cover the essential components for configuring Maestro:
+
+- [Elasticsearch](/documentation/score/installation/configuration/profiles): 	Set the index mapping and configure connection parameters and error handling mechanisms.
```
[the remark this answers is not in the document]
Comment (MitchellShiell) [reply markdown/documentation/maestro/installation/configuration/index.md:7]: Weird, I'll check all links and update

**Review by leoraba — APPROVED** [review]
Looks good. Thanks!

**Pull request merged**